augur/application/db/models/augur_data.py

```diff
@@ -3197,7 +3201,8 @@ class PullRequestMessageRef(Base):
             onupdate="CASCADE",
             deferrable=True,
             initially="DEFERRED",
-        )
+        ),
+        nullable=False,
```

Comment:
This seems to make a change to the nullable status of another table. Was there a particular reason for this change? How would you recommend existing production databses, some with tens or hundreds of thousands of repos perform this migration if their tables have null values in them?

Reply:
it looks like this particular line is affecting a class called `PullRequestMessageRef` - is that one that you added?